Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-AKS-cluster/aks-cluster.tf

```diff
@@ -1,38 +1,90 @@
 
+#############
+# LOCALS #
+#############
+
+/*
+
```

Comment:
Please add comment indicating that if Blue/Green scenario is not used then this code can be simplified by specifying only one cluster.

Reply:
fixed